pkg/apicast/image.go

```diff
@@ -1,6 +1,6 @@
 package apicast
 
-const currentImageVersion = "quay.io/3scale/apicast:nightly"
+const currentImageVersion = "quay.io/3scale/apicast:v0.1.0"
```

Comment:
Changed "default" image for apicast in this branch to v0.1.0

Reply:
see comment above